templates/taxi/manufacturer_list.html

```diff
@@ -8,6 +9,11 @@ <h1>
     </a>
   </h1>
 
+  <form action="" method="get" class="form-inline">
+    {{ manufacturer| crispy }}
+    <input type="submit" value="🔍" class="btn btn-secondary">
+  </form>
```

Comment:
Duplicate of code. Please create and include html for this

Reply:
okay redone, but reduplication in includes is better, or i miss smth? or is there an opportunity to move let's say manufacturer in variable?